tests/diff/float_values_test.go

```diff
@@ -61,6 +63,87 @@ func TestFloatValues(t *testing.T) {
 
 				_, err := db.Collection("insert-"+name).InsertOne(ctx, tc.doc)
 
+				t.Run("FerretDB", func(t *testing.T) {
+					assertEqualError(t, tc.expected, err)
+				})
+
+				t.Run("MongoDB", func(t *testing.T) {
+					require.NoError(t, err)
+				})
+			})
+		}
+	})
+
+	t.Run("Update", func(t *testing.T) {
+		t.Parallel()
+
+		// to set UpdateOptions.Upsert.
+		boolPointer := func(b bool) *bool {
```

Comment:
I see we don't use `pointer` library in dance, but I think it's ok to introduce it because it might be needed in future too. See how `pointer.To` is used in `ferretdb`.